packages/contracts-bedrock/scripts/getting-started/config.sh

```diff
@@ -65,8 +65,8 @@ config=$(cat << EOL
   "l1FeeVaultWithdrawalNetwork": 0,
   "sequencerFeeVaultWithdrawalNetwork": 0,
 
-  "gasPriceOracleOverhead": 2100,
-  "gasPriceOracleScalar": 1000000,
+  "gasPriceOracleOverhead": 0,
+  "gasPriceOracleScalar": "0x010000000000000000000000000000000000000000000000000c5fc500000558",
```

Comment:
Given https://github.com/ethereum-optimism/optimism/pull/10226 which correctly sorts out the scalar issues

Reply:
@tynes Maybe we fix this properly once your PR is in, but for now use the encoded scalar value? Otherwise we leave the getting started script in an inconsistent state.

Reply:
We can even put the fix into https://github.com/ethereum-optimism/optimism/pull/10226, then things are being updated atomically in a way that makes sense.

Reply:
I would prefer that we revert the diff on the scalar typing as it will break tooling then this PR is good to be merged. https://github.com/ethereum-optimism/optimism/pull/10226 has been cleaned up and is ready for review

Reply:
Like this?
```suggestion
  "gasPriceOracleScalar": 452312848583266388373324160190187140051835877600158453279134670530344387928,
```

Reply:
if you rever the diff on line 69 then this is good to merge